src/rules/enforce-boolean-naming-prefixes.ts

```
@@ -399,6 +394,73 @@ export const enforceBooleanNamingPrefixes = createRule<Options, MessageIds>({
+          'assertresult',
+          'assertresponse',
+          'assertconfig',
+          'assertsettings',
+          'assertoptions',
+          'assertformat',
+          'assertschema',
+          'assertstructure',
+        ];
+
+        const lowerCalleeName = calleeName.toLowerCase();
+        const isValidationAssert = validationAssertPatterns.some(pattern =>
```

ESLint

[error] Replace `pattern` with `(pattern)`

(prettier/prettier)